spec/display_spec.rb

```diff
@@ -28,20 +28,30 @@
       end.to output('Choose a position from 1-9: ').to_stdout
     end
 
-    it 'returns the users move as an integer' do
+    it 'returns the users move' do
```

Comment:
This is an example of why I prefer `"` to `'`, because the test name should be "returns the user's move"

Reply:
Thanks, Matt